extension/third_party_attribute_extension.go

```diff
@@ -0,0 +1,69 @@
+package extension
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/getyoti/yoti-go-sdk/v2/attribute"
+)
+
+const (
+	thirdPartyAttributeExentionTypeConst = "THIRD_PARTY_ATTRIBUTE"
+)
+
+// ThirdPartyAttributeExtensionBuilder is used to construct a ThirdPartyAttributeExtension
+type ThirdPartyAttributeExtensionBuilder struct {
+	extension ThirdPartyAttributeExtension
+}
+
+// ThirdPartyAttributeExtension is an extension representing the issuance of a third party attribute
+type ThirdPartyAttributeExtension struct {
```

Comment:
Might be a bit clearer if we split up the `ThirdPartyAttributeExtension` and `ThirdPartyAttributeExtensionBuilder` into separate files? I think we have that separation with the other builders in the Go SDK